ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/router-ospf.yml

```diff
@@ -70,10 +70,13 @@ router_ospf:
       redistribute:
         static:
           route_map: rm-ospf-static
+          include_leaked: true
         connected:
           route_map: rm-ospf-connected
+          include_leaked: true
```

Comment:
can we also test include_leaked true without route-map? seems to be accepted by EOS as well
```
Leaf1(config-router-ospf)#redistribute bgp include leaked
Leaf1(config-router-ospf)#show active
router ospf 666
   redistribute bgp include leaked
```